mindmeld/auto_annotator.py

```diff
@@ -835,5 +895,282 @@ def valid_entity_check(self, entity):
         return Entity.is_system_entity(entity) or entity in self.supported_entity_types
 
 
+class NoTranslationDucklingAnnotator(Annotator):
+    """Custom Annotator class used to generate annotations."""
+
+    def __init__(self, app_path, config=None):
```

Comment:
Its not very clear to me why the spacy annotator needs the language and locale in the constructor but the other annotator classes do not.

Reply:
Consistent now